src/betterproto/plugin/models.py

```diff
@@ -279,6 +279,10 @@ def proto_name(self) -> str:
     def py_name(self) -> str:
         return pythonize_class_name(self.proto_name)
 
+    @property
+    def py_name_as_field(self) -> str:
```

Comment:
looks like this isn't used either actually?

Reply:
Not anymore since 5bbe19ab. Nice catch. Good thing you're looking over this. I completely forgot.